instrumentation/netty/netty-3.8/javaagent/src/test/groovy/Netty38ClientTest.groovy

```diff
@@ -70,17 +72,23 @@ class Netty38ClientTest extends HttpClientTest<Request> implements AgentTestTrai
 
   @Override
   void sendRequestWithCallback(Request request, String method, URI uri, Map<String, String> headers, AbstractHttpClientTest.RequestResult requestResult) {
+    // TODO: context propagation into callbacks is not implemented
```

Comment:
I guess this would need to be corrected with `async-http-client` instrumentation, but we don't instrument `async-http-client` 1.8.

Reply:
```suggestion
    // TODO: context is not automatically propagated into callbacks
```